test/testdrive/timestamps-kafka-avro-multi.td

```diff
@@ -7,7 +7,7 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-$ set-sql-timeout duration=30s
+$ set-sql-timeout duration=60s
```

Comment:
I was running into ci errors over a few reruns that I couldn't reproduce on my laptop. I think its because the timestamper only checks for updated kafka metadata every 30 seconds for rt topics

still strange that this would start to flake now

Reply:
i ran this test several times on my laptop with varying numbers of workers in dev mode - got one flake